nemo/collections/nlp/models/language_modeling/megatron/gpt_model.py

```diff
@@ -321,8 +323,10 @@ def state_dict_for_save_checkpoint(self, destination=None, prefix='', keep_vars=
             )
         return state_dict_
 
-    def load_state_dict(self, state_dict, strict=True):
+    def load_state_dict(self, state_dict, strict=True, bypass=False):
         """Customized load."""
+        if bypass:
+            return super(GPTModel, self).load_state_dict(state_dict, strict=strict)
```

Comment:
This seems hacky. Can you detect when you need to bypass and just bypass then instead of changing the signature?

Reply:
I have removed the bypass flag entirely